tools/build-tools/data/layerInfo.json

```diff
@@ -164,7 +164,8 @@
             },
             "Framework": {
                 "packages": [
-                    "@fluid-internal/client-api"
+                    "@fluid-internal/client-api",
+                    "@fluid-experimental/experimental-fluidframework"
```

Comment:
I believe layer check needs to go in first, then you rev the layer check version on the root, then you check in this change.